cosmos/kopi.json

```diff
@@ -0,0 +1,91 @@
+{
+  "chainId": "luwak-1",
+  "chainName": "kopi",
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/kopi/chain.png",
+  "rpc": "https://kopi-rpc.polkachu.com",
+  "rest": "https://kopi-api.polkachu.com",
+  "nodeProvider":{
+     "name":"Polkachu",
+     "email":"hello@polkachu.com",
+     "website":"https://polkachu.com/"
+  },
+  "bip44": {
+     "coinType": 118
+  },
+  "bech32Config": {
+     "bech32PrefixAccAddr": "kopi",
+     "bech32PrefixAccPub": "kopipub",
+     "bech32PrefixValAddr": "kopivaloper",
+     "bech32PrefixValPub": "kopivaloperpub",
+     "bech32PrefixConsAddr": "kopivalcons",
+     "bech32PrefixConsPub": "kopivalconspub"
+  },
+  "currencies": [
+     {
+        "coinDenom": "XKP",
+        "coinMinimalDenom": "ukopi",
+        "coinDecimals": 6,
+        "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/kopi/chain.png"
+     },
+     {
+        "coinDenom": "KUSD",
+        "coinMinimalDenom": "ukusd",
+        "coinDecimals": 6,
+        "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/kopi/kUSD.png"
+     },
+     {
+        "coinDenom": "CKUSD",
+        "coinMinimalDenom": "uckusd",
+        "coinDecimals": 6,
+        "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/kopi/kUSD.png"
+     },
+     {
+        "coinDenom": "CUSDC",
+        "coinMinimalDenom": "ucusdc",
+        "coinDecimals": 6,
+        "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/noble/uusdc.png"
```

Comment:
Please add all the images files to `images/luwak`, instead of images/noble or images/injective, as only the same image directory name to a JSON file name is acceptable.